assignments/stretch-function-conversion.js

```diff
@@ -1,23 +1,16 @@
 // Take the commented ES5 syntax and convert it to ES6 arrow Syntax
```

Comment:
I love that you did this stretch. 

This will make understanding and reading arrow functions and what it means in ES5 so much easier. I want everyone to go back and do this stretch. Great job